benchmarks/benchmarks/bench_random.py

```diff
@@ -34,11 +34,11 @@ class Randint(Benchmark):
 
     def time_randint_fast(self):
         """Compare to uint32 below"""
-        np.random.randint(0, 2**30, size=10**5)
+        np.random.randint(low=0, high=2**30, size=10**5)
```

Comment:
@juliantaylor : The tests are <a href="https://github.com/gfyoung/numpy/blob/randint_no_low/numpy/random/tests/test_random.py#L211">here</a>